dev/io.openliberty.webcontainer.servlet.6.0.internal/src/io/openliberty/webcontainer60/session/impl/SessionAffinityManagerImpl60.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.webcontainer60.session.impl;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.logging.Level;
+
+import com.ibm.ws.session.SessionContext;
+import com.ibm.ws.session.SessionManagerConfig;
+import com.ibm.ws.session.utils.LoggingUtil;
+import com.ibm.ws.webcontainer.session.impl.SessionAffinityManagerImpl;
+import com.ibm.wsspi.session.IStore;
+import com.ibm.wsspi.session.SessionAffinityContext;
+import com.ibm.wsspi.webcontainer.servlet.IExtendedResponse;
+
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.SessionCookieConfig;
+import jakarta.servlet.http.Cookie;
+
+public class SessionAffinityManagerImpl60 extends SessionAffinityManagerImpl {
+
+    private static final String methodClassName = "SessionAffinityManagerImpl60";
+
+    public SessionAffinityManagerImpl60(SessionManagerConfig smc, SessionContext sctx, IStore istore) {
+        super(smc, sctx, istore);
+
+        if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled() && LoggingUtil.SESSION_LOGGER_CORE.isLoggable(Level.FINE)) {
+            LoggingUtil.SESSION_LOGGER_CORE.logp(Level.FINE, methodClassName, methodClassName, "Clone ID of this server=" + _cloneID);
```

Comment:
should the second `methodClassName` actually be the method name and not the class name?

Reply:
they are the same since it is the constructor.